presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUpdateablePageSource.java

```diff
@@ -125,6 +174,10 @@ public boolean isFinished()
     @Override
     public Page getNextPage()
     {
+        // strategy
+        // 1. retrieve rows from delegate
+        // 2. apply delete filter to page
+        // 3. transform page for update (turn flat blocks into nested update row)
```

Comment:
I would convert this into a method level comment and use full sentences, proper punctuation and capitalization